9-Tools/configfile_generator/ConstructAccess.py

```diff
@@ -286,48 +421,34 @@ def case7():
 	for elem in args:
 		limit_resources(lpath,elem)
 
-
-# Syntax "python ConstructAccess.py help " 
-# This print all commands syntax for you
+# Syntax python ConstructAccess.py -m addRoles -u sa_user_name -s sa_user_namespace -n namespace_1 -n namespace_2 -a access_kind_for_namespace_2 
 def case8():
-	print(bcolors.FAIL + "All commands: with python ConstructAccess.py plus these commands below\n" + bcolors.ENDC)
-	print(ErrorSyntax.get("create"))
-	print(ErrorDescription.get("create"))
-	print(ErrorSyntax.get("createEx"))
-	print(ErrorDescription.get("createEx"))
-	print(ErrorSyntax.get("createCustomRole"))
-	print(ErrorDescription.get("createCustomRole"))
-	print(ErrorSyntax.get("createExCustomeRole"))
-	print(ErrorDescription.get("createExCustomeRole"))
-	print(ErrorSyntax.get("recreate"))
-	print(ErrorDescription.get("recreate"))
-	print(ErrorSyntax.get("limit"))
-	print(ErrorDescription.get("limit"))
+	sa_username = optsdict['uname'][0]
+	sa_user_namespace_name = optsdict['unsname'][0]
+	ns_list = optsdict['nsname']
+	akind_list = optsdict['akind']
+	addRoles(sa_username,sa_user_namespace_name,ns_list,akind_list)
+
 
 # all flags and their shorthands, 
 # --nsname -n
 # --uname -u
+# --unsname -uns
 # --akind -a
 # --rpath -r
 # --lpath -l
 if __name__ == '__main__':
-	cases = {"merge":case1,"create":case2,"createEx":case3,"createCustomRole":case4,
-	"createExCustomRole":case5,"recreate":case6,"limit":case7,"help":case8}
-	try:
-		opts, args = getopt.getopt(sys.argv[2:], 'n:u:a:r:l:',['nsname=', 'uname=','akind=','rpath=','lpath='])
-	except getopt.GetoptError:
-		print(bcolors.FAIL + "Input Error" + bcolors.ENDC)
-		sys.exit(2)
-	optsdict = dict(opts)
-	if sys.argv[1] not in cases:
+	cases = {"help":case0,"merge":case1,"create":case2,"createEx":case3,"createCustomRole":case4,
+	"createExCustomRole":case5,"recreate":case6,"limit":case7,"addRoles":case8}
```

Comment:
Method `caseN` is not very readable, could you please rename them? For example, `case0` could be `help()` instead.

Reply:
how about I call cases as `methods` ? . If I understand correctly the problem is the keys in cases are not cases* . :)

Reply:
I mean you have a series of methods, like `case0()`, `case1()`, etc. Why not directly name them as `help()`, `merge()`?

Reply:
Ah I see what you mean now :) . I'll see what I can do.

Reply:
It is now refactored :) .